agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/AiSemanticAttributes.java

```diff
@@ -21,15 +21,18 @@
 
 package com.azure.monitor.opentelemetry.exporter.implementation;
 
+import static io.opentelemetry.api.common.AttributeKey.longKey;
+import static io.opentelemetry.api.common.AttributeKey.stringKey;
+
 import io.opentelemetry.api.common.AttributeKey;
 
 public final class AiSemanticAttributes {
 
   public static final AttributeKey<String> OPERATION_NAME =
-      AttributeKey.stringKey("applicationinsights.internal.operation_name");
+      stringKey("applicationinsights.internal.operation_name");
```

Comment:
we don't typically static import the AttributeKey methods upstream in OpenTelemetry (there's a bit of guidance upstream which could probably use updating here: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/style-guideline.md#static-imports)

Reply:
[check here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/65717dae3e7d7ee2708e5e3a76744da82333ee2e/instrumentation/hystrix-1.4/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/hystrix/ExperimentalAttributesExtractor.java#:~:text=import%20static%20io.opentelemetry.api.common.AttributeKey.booleanKey%3B)
i think that list needs to be updated.

Reply:
ah, ya, looks like its inconsistent upstream, mostly it's not statically imported, but looks like it is in a few places, so no worries, can keep as is if you prefer